tests/test_enforce.py

```diff
@@ -95,3 +103,62 @@ def nonfinal_classmethod(cls):
 
         self.assertNotEqual(ClassMethodOverrider.nonfinal_classmethod(),
                             Enforcing.nonfinal_classmethod())
+
+    def test_is_compatible_when_compatible(self):
+        def foo(x, /, y: str, *args, z: int, **kwargs) -> object:
+            pass
+
+        def bar(x, /, y: object, a: float, *, b: bytes, z: int, **kwargs) -> str:
+            pass
+
+        is_compatible(foo, bar)
```

Comment:
The naming could be improved a bit so that it is clear which is super and which is sub.